content/umdctf_2024/tcache.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Tcache 
+date: 2024-04-17T14:23:28+03:00
+description: Information about tcache 
+author: PineBel
+tags:
+    - pwn
+
+----
+## Tcache
+
+Tcache (per thread cache) was added in glibc 2.26 to make heap allocation more efficient, and basically each thread will get it s own bin of small chunks. 
```

Comment:
its*
long sentence: more efficient. Each thread ...
also what does _own bin of small chunks_ mean?